src/Resources/skeleton/authenticator/EmptyAuthenticator.tpl.php

```diff
@@ -8,46 +8,79 @@
 use Symfony\Component\Security\Core\User\UserInterface;
 use Symfony\Component\Security\Core\User\UserProviderInterface;
 use Symfony\Component\Security\Guard\AbstractGuardAuthenticator;
+use Symfony\Component\Security\Guard\Token\GuardTokenInterface;
 
 class <?= $class_name ?> extends AbstractGuardAuthenticator
 {
-    public function supports(Request $request)
+    /**
+     * {@inheritdoc}
+     */
+    public function supports(Request $request): bool
     {
-        // todo
+        // TODO: implement method
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
I dislike these `{@inheritdoc}` so much 😢  Isn't PhpStorm smart enough to do that without adding this?

Reply:
I would be nice indeed (I don't use PhpStorm, and I don't care about theses inheritdoc)